### DISCUSSION TITLE
Ad block and track Block

### DISCUSSION THREAD

**Discussion by faresmuontaser** [zen-browser/desktop#3375] [Ideas]
It would be nice if we had built in ad blocker

**Comment by princess-wawa** [comment]
Firefox has good AdBlocker addons

**Comment by dnut** [comment]
Strongly disagree. Spending time developing this is a complete waste of resources when uBlock Origin already exists.

**Reply by faresmuontaser** [reply]
Bruh I forgot about it☠️
I'm just suggesting not asking so it's not an argument
